apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -81,6 +83,7 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
             }
         }
 
+        context.setContext(STOP_SPAN_FLAG);
         AbstractSpan span = ContextManager.createExitSpan(operationName, peer);
```

Comment:
```suggestion
        AbstractSpan span = ContextManager.createExitSpan(operationName, peer);
        context.setContext(STOP_SPAN_FLAG);
```

Put the flag after Created, this should be safer.